### PR TITLE
Clarify examples in README.

### PR DESCRIPTION
**Pull request by paragonie-scott**
See #46

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,6 +16,7 @@ Let's refactor the following legacy insecure code snippet to prevent SQL injecti
 
 ```php
 $query = mysql_query(
+    /*                                         v- Don't do this. -v                     */
     "SELECT * FROM comments WHERE blogpostid = {$_GET['blogpostid']} ORDER BY created ASC"
```
Comment (shadowhand) [inline README.md]: I would say this example should be completely eliminated from the readme. People are apt to copy-paste things, even when there's a comment that says it shouldn't be done.

Comment (paragonie-scott) [reply README.md]: How else would your propose educating users in the process?

Comment (shadowhand) [reply README.md]: IMO, just state the problem without providing example code.

**Comment by paragonie-scott** [comment]
You're right. Better to just remove it.

**Pull request closed without merging**